packages/emotion/typings/emotion.d.ts

```diff
@@ -0,0 +1,47 @@
+export type Interpolation = string | number | boolean | null | undefined | _Interpolation1 | _Interpolation2 | _Interpolation3;
+
+// HACK: See https://github.com/Microsoft/TypeScript/issues/3496#issuecomment-128553540
+interface _Interpolation1 extends Record<string, Interpolation> {}
+interface _Interpolation2 extends Array<Interpolation> {}
+interface _Interpolation3 {
+  (): Interpolation;
+}
+
+export type CreateStyles<TRet> = ((value: Interpolation) => TRet)
+  & ((strings: TemplateStringsArray, ...vars: Interpolation[]) => TRet);
+
+export interface StylisUse {
+  // TODO: Make this more precise than just Function
+  (plugin: Function | Function[] | null): StylisUse;
+}
+
+export interface StyleSheet {
+  inject(): void;
+  speedy(bool: boolean): void;
+  insert(rule: string, sourceMap: string): void;
+  flush(): void;
+}
+
+export const sheet: StyleSheet;
+
+export const useStylisPlugin: StylisUse;
+
+export const inserted: Record<string, boolean | undefined>;
+
+export const registered: Record<string, string | undefined>;
+
+export function flush(): void;
+
+export function values(cls: string, vars: Interpolation[]): string;
```

Comment:
This export doesn't exist anymore, I'm guessing this is from the other TypeScript PR?

Reply:
I think so, I removed it.